debug/expressions.html

```diff
@@ -0,0 +1,86 @@
+            }
+        }
+    },
+    layers: [
+        {
+            id: 'circle',
+            type: 'circle',
+            source: 'geojson',
+            paint: {
+                'circle-radius': {
+                    type: 'expression',
+                    expression: [
+                        '*',
+                        [ '^', 2, {ref: 'map', key: 'zoom'} ],
+                        0.125,
+                        [
+                            '+',
+                            20,
+                            {ref: 'feature', key: 'x'},
+                            {ref: 'feature', key: 'y'}
+                        ]
+                    ]
```

Comment:
That's so nice!  What do `map.zoom`, `feature.x`, etc. end up looking like in the resulting NSExpression object?